src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -199,138 +330,6 @@ protected Task TestSmartTagGlyphTagsAsync(string initialMarkup, ImmutableArray<s
             }
         }
 
-        protected async Task TestAddDocumentInRegularAndScriptAsync(
```

Comment:
Split into a separate partial file which is not included in CodeStyle test utilities project.